### PR TITLE
Add contrib

### PR DESCRIPTION
**Pull request by AmailP**

### CODE REVIEW

**Comment by skhomuti** [comment]
Tests do not work on the new version. Is there any way to find a backwards incompatible changes in new versions of an idea? I did not find such information in https://blog.jetbrains.com/idea/category/eap-releases/

**Comment by AmailP** [comment]
I have asked a question in the Plugin Development community:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360006388540-Exception-for-plugin-in-latest-EAP-Action-ForceStepInto-doesn-t-override-anything

**Comment by skhomuti** [comment]
same on newer EAP versions
image

**Comment by AmailP** [comment]
> same on newer EAP versions python-ce
> image
Good catch... I have answered on the support forum.

File: build.gradle [hunk 1]
```diff
@@ -27,7 +27,7 @@ def pythonPluginForVersion = [
         "IC-183.4284.36": "PythonCore:2018.3.183.4284.36",
         "IC-191.6183.87": "PythonCore:2019.1.191.6183.53",
         "IC-192.5728.98": "PythonCore:2019.2.192.5728.98",
-        "LATEST-EAP-SNAPSHOT": "PythonCore:2019.2.192.5728.98",
+        "IC-193.4386.10-EAP-SNAPSHOT": "PythonCore:193.4386.10",
```
Comment (skhomuti) [inline build.gradle]: Newest version works fine. Let's leave these versions.
```suggestion
        "193.4697-EAP-CANDIDATE-SNAPSHOT": "PythonCore:193.4697.15",
```

File: build.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@ repositories {
 
 version = gitVersion()
 
-def intellijVersion = System.getenv().getOrDefault("IDEA_VERSION", "IC-192.5728.98")
+def intellijVersion = System.getenv().getOrDefault("IDEA_VERSION", "IC-193.4386.10-EAP-SNAPSHOT")
```
Comment (skhomuti) [inline build.gradle]:
```suggestion
def intellijVersion = System.getenv().getOrDefault("IDEA_VERSION", "193.4697-EAP-CANDIDATE-SNAPSHOT")
```

File: .travis.yml [hunk 3]
```diff
@@ -6,19 +6,23 @@ scala:
...
+# Usually the list contains least one for each major version between
+# "sinceBuild" and "untilBuild" (from build.gradle).
+
 env:
   - IDEA_VERSION=IC-2017.3.5
-  - IDEA_VERSION=IC-2018.1.6
   - IDEA_VERSION=IC-183.4284.36
-  - IDEA_VERSION=IC-191.6183.87
   - IDEA_VERSION=IC-192.5728.98
+  - IDEA_VERSION=IC-193.4386.10-EAP-SNAPSHOT
```
Comment (skhomuti) [inline .travis.yml]:
```suggestion
  - IDEA_VERSION=193.4697-EAP-CANDIDATE-SNAPSHOT
```

**Pull request merged**